lfx-mentorship/2022/03-Sept-Nov/project_ideas.md

```diff
@@ -187,4 +187,13 @@ a new and popular computing platform, Volcano should integrate with it.
 - Description: Users have no ways to update log level of Volcano components now, which is difficult to track bugs especially in the production environment. Your
 task is to achieve it.
 - Mentor(s): [william-wang](https://github.com/william-wang)
-- Issue: <https://github.com/volcano-sh/volcano/issues/2430>
\ No newline at end of file
+- Issue: <https://github.com/volcano-sh/volcano/issues/2430>
+
+#### Thanos
+
+##### Support for tenant-specific external labels in Thanos Receiver
```

Comment:
Is this a placeholder title?

Reply:
Ah sorry, it was a duplicated title. I've removed one of them :)